plexapi/video.py

```diff
@@ -32,6 +33,8 @@ def _loadData(self, data):
         """ Load attribute values from Plex XML response. """
         self._data = data
         self.listType = 'video'
+        self.guid = data.attrib.get('guid')
```

Comment:
Im not sold on the changed to the video class. This is getting reusing many places. Would it be better for subclass used own class for live tv/recordings) ? Im not sure.

Reply:
Hasn't given me an issue, but I can subclass it just to make sure

Reply:
Strike that, the TAG and type are the same for Live videos and non-live videos, so I can't register a new Plex object. Plex doesn't make a distinction between them, so I think it's safe to just add the guid and live attributes, and the record method (checks if video is live first)